packages/mui-material/src/Button/Button.js

```diff
@@ -345,12 +345,323 @@ const ButtonEndIcon = styled('span', {
   ],
 }));
 
+export const ButtonRootMD3 = styled(ButtonBase, {
```

Comment:
These are just copied from the previous @mui/material-next package, they would need to be converted to support Pigment CSS.